File: src/server/InvestmentApp-Server/Models/Experts/ExpertIndustry.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;
using InvestmentApp.Models.Industries;

namespace InvestmentApp.Models.Experts;

public class ExpertIndustry
{
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public Guid Id { get; set; }

    public Industry Industry { get; set; }

    public Guid IndustryId { get; set; }

    public Expert Expert { get; set; }

    public Guid ExpertId { get; set; }

    public double Rate { get; set; }
}

File: src/client/src/app/models/expert-industry.ts
import { DataTableRow } from ".";
import { Guid } from "guid-typescript";
import { Criteria } from "./criteria";
import { Industry } from "./industry";
import { Expert } from "./expert";

export interface ExpertIndustry {
    id: Guid
    industry: Industry;
    industryId: Guid;
    expert: Expert;
    expertId: Guid;
    rate: number;
}

export interface ExpertIndustryRow extends DataTableRow {
    id: string,
    internalId: string;
    industry: string;
    industryId: string;
    expert: string;
    expertId: string;
    rate: number;
}
